client/src/components/dashboard/Add.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+// need connect function to be able to connect to store from Provider
+import {connect} from 'react-redux';
+
+import {dashDefault} from '../../actions/dashActions';
+
+import moment from 'moment';
+
+class Container extends React.Component {
+    constructor(props) {
+        super(props);
+        // this.setDashDefault = this.setDashDefault.bind(this);
+    }
+
+    submitForm = (e) => {
+        e.preventDefault();
+        const formData = new FormData(document.getElementById('form-add'));
+        // this.props.setDashDefault();
+        let obj = {};
+        for(let data of formData.entries()) {
```

Comment:
Is there a reason why you chose to get the form data directly from the DOM rather than use controlled components (the more "React" way)?
Just curious to hear your thought process.

Reply:
hmmm i am not sure how to do it in React, and i'm way more familiar with JS than React. FormData also targets the form elements themselves neatly and in one-go, no matter if they're nested in divs and whatnot.

Reply:
Alright, fair enough. For future reference though: https://stackoverflow.com/a/42522792